proposals/003-installer/installer-guidelines.md

```diff
@@ -0,0 +1,79 @@
+# Installer Guidelines
+
+## 0. Objectives
+
+We want people to use Haskell, which means the path to installing Haskell for first-time users
+should be short and easy, and it should yield an environment that is easy to use with the supporting
```

Comment:
This is vague and should describe the user experience. What is easy? For which type of user?

There has already been strong disagreements on whether "interactive" is considered easy or not.

Reply:
I think this needs to be left as an issue for the collective judgement of users.  My strong suspicion is that there's a use case for both interactive and non-interactive installation, and that an ideal install tool would provide both options while installing the same a well-known and standardized environment regardless of the choice of UX.

Reply:
Yeah, `ghcup` does this today. But I think it's not useful to have such fine-grained technical requirements. It sounds like forcing ideas and requirements on projects through a formal proposal, instead of *opening issues on the respective issue trackers*, which HF has never done.

In general, I dislike the tone of this proposal. Maintainers don't work for the HF (at least I don't), they work for the community. And that community can already reach out to them and get their ideas and needs discussed.

So what do we need this proposal really for?